releasenotes/notes/enable-windows-parallel-map-b33122a2122a95d7.yaml

```diff
@@ -0,0 +1,55 @@
+---
+features:
+  - |
+    When running on Windows parallel execution with
+    :func:`~qiskit.tools.parallel_map` (primarily with
+    :func:`~qiskit.compiler.transpile` and :func:`~qiskit.compiler.assemble`)
+    can now be enabled. To do this you can either set ``parallel = True`` in
+    a user config file, or set the ``QISKIT_PARALLEL`` environment variable to
+    ``TRUE``.
+issues:
+  - |
+    On Windows systems when parallel execution is enabled for
+    :func:`~qiskit.tools.parallel_map` (for example with
+    :func:`~qiskit.compiler.transpile` and :func:`~qiskit.compiler.assemble`)
+    may not work when caleld from a script running outside of a
+    ``if __name__ == '__main__':`` block. This is due to how Python launches
+    parallel processes on Windows. If ``RuntimeError`` or ``AttributeError``
+    are raised by scripts that are directly calling parallel_map() or when
+    calling a function that uses it internally with Windows and parallelism
+    enabled embedding the script calls inside ``if __name__ == '__main__':``
```

Comment:
I found this sentence a little hard to parse. The issue can occur when a runtime or attribute error is raised from code called through parallel_map, or code calling parallel_map?

Reply:
Yeah it was worded poorly, I was trying to say anywhere parallel_map is used which included things calling it internally, like transpile(). I've updated it in: https://github.com/Qiskit/qiskit-terra/pull/3547/commits/44e5ebceed55a70a52ae9bfc321e72e9a4ba486e to try and clean it up.